builder/bin/image_gen.rs

```diff
@@ -19,6 +19,14 @@ fn main() {
                 .value_parser(value_parser!(PathBuf)),
         )
         .arg(arg!(--"fw" [FILE] "FW bundle image").value_parser(value_parser!(PathBuf)))
+        .arg(
+            arg!(--"fmc-svn" [FILE] "Security Version Number of FMC firmware image")
```

Comment:
```suggestion
            arg!(--"fmc-svn" [VALUE] "Security Version Number of FMC firmware image")
```

nit: I think `[FILE]` could cause confusion when reading the help text.

Reply:
Done